tests/ActivityloggerTest.php

```diff
@@ -243,4 +244,59 @@ public function it_returns_an_instance_of_the_activity_log_after_logging_when_us
 
         $this->assertInstanceOf($activityClassName, $activityModel);
     }
+
+    /** @test */
+    public function it_uses_any_guards_user_used()
+    {
+        Auth::logout();
```

Comment:
i wanna make sure there is no log in user from other test... well if this is not persisted per test then its not needed then